Clients/Xamarin.Interactive.Client.Web/ClientApp/boot.tsx

```diff
@@ -5,7 +5,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-import './boot.scss'
+import 'es6-shim';

```

Comment:
Is this shim a no-op for real browsers or does it override native implementations of stuff?

Reply:
It looks like it performs a variety of tests on the implementations of these APIs, because some older browsers have bad implementations. See https://github.com/paulmillr/es6-shim/blob/master/es6-shim.js#L864, for example.

We could import this conditionally.

Reply:
Nah, I don't mind it if native implementations shine through.